scripts/deploy_contracts.js

```diff
@@ -18,9 +18,9 @@ const gamedata = require(`../gamedata/gamedata.json`)
 
 let web3;
 let ethernaut;
-let deployData;
 
-const PROMPT_ON_DEVELOP = true
+const PROMPT_ON_DEVELOP = false
```

Comment:
I would prefer to change the behaviour in a separate PR.

Reply:
My bad, I overlooked this one. I will revert this change.